src/frontend/containers/block/note.tsx

```diff
@@ -1,31 +1,21 @@
 import { upsertOne } from "@db/index";
+import * as queries from "@db/queries";
 import * as dbTypes from "@db/types";
+import { uniqueId } from "@frontend/utils";
+
 import React from "react";
 import {
   Observable,
   of, // concatMap,
   /// delay
 } from "rxjs";
-import { v4 as uuidv4 } from "uuid";
 
 export const Component = (props: { db: dbTypes.LoadedDb; block: dbTypes.BlockNote }) => {
   const { db, block } = props;
   const { body } = block;
 
   const updateText = (text: string) => {
-    // This doesn't work because od all the _rev, _meta info
-    //    const newBlock = { ...block, body: text }
-
-    // this does work for a traditional upsert: see issue #41
-    const newBlock: dbTypes.BlockNote = {
-      body: text,
-      // @ts-ignore
-      document_type: "block" as dbTypes.DbDocumentType,
-      id: block.id,
-      state: block.state,
-      type: "note",
-    };
-    upsertOne(db.instance, newBlock);
+    queries.merge(db.instance, block.id, { body: text });
```

Comment:
this shows off the simplicity of the new merge function.

Reply:
The downside is, changes are not type checked.  If we change the implementation of one of the Document types, we will have to search the codebase for changes.

Reply:
I tried to address this in [776e77b](https://github.com/dw-innovation/dw-kid2-graph-frontend-prototype/pull/56/commits/776e77be8dfa6fc4a40852fde4a255c05d4e6d2a)